ucm2/sof-soundwire/RT711.conf

```diff
@@ -0,0 +1,44 @@
+# Use case Configuration for sof-soundwire card
+
+If.RT711 {
+	Condition {
+		Type String
+		Haystack "${CardComponents}"
+        	Needle "hs:rt711" 
+	}
+	True {
+
+		SectionDevice."Headphone" {
+			Comment	"Headphone"
+
+			EnableSequence [
+				cset "name='rt711 DAC Surr Playback Volume' 87"
```

Comment:
we need to add "cset "name='Headphone Switch' on" here, or there is no output in headphone when switch speaker to headphone in sound setting